source/elements/oneMKL/source/domains/spblas/data_types/format-descriptions.rst

```diff
@@ -0,0 +1,152 @@
+.. SPDX-FileCopyrightText: 2024 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_sparse_format_descriptions:
+
+Sparse storage formats
+======================
+
+There are a variety of matrix storage formats available for representing sparse
+matrices. Two popular formats are the coordinate (COO) format, and the
+Compressed Sparse Row (CSR) format.
+
+In this specification, "non-zero" elements or "non-zero" entries refer to
+explicitly defined elements or entries which may take any value supported by the
+the `:ref:data type<onemkl_sparse_supported_types>`. Undefined elements or
+entries are implicitly zeros.
+
+.. container:: section
+
+   .. _onemkl_sparse_coo:
+
+   .. rubric:: COO
+
+   The COO format is the simplest sparse matrix format, represented by three
+   arrays, ``row_ind``, ``col_ind`` and ``val``, and an ``index`` parameter. The
+   ``i``-th defined element in the sparse matrix is represented by its row
+   index, column index, and value, that is, (``row_ind[i]``, ``col_ind[i]``,
+   ``val[i]``). The entries need not be in a sorted order.
```

Comment:
```suggestion
   ``val[i]``). The entries need not be in a sorted order, though performance of Sparse BLAS oeprations may be improved if there it is sorted in some logical way, for instance by row index and then column index subordinate to each row set."
```
This seems like a perfect place to mention that while sortedness is not required, it can have a performance impact.  What do you think ?

Reply:
Sure I've added your suggestion in https://github.com/uxlfoundation/oneAPI-spec/pull/522/commits/ebc779ced269cf6d6b429c1897b79d071fe79c06 with a minor correction and formatted to split the line.